role-based-access-control.md

```diff
@@ -278,10 +278,14 @@ SET ROLE ALL EXCEPT 'app_read'
 SET DEFAULT ROLE ALL TO 'rw_user1'@'localhost';
 ```
 
-用 `rw_user1@localhost` 登陆后：
+用 `rw_user1@localhost` 登陆，TiDB中无法通过 `localhost` 登陆，需要在 tidb-server 机器上指定127.0.0.1登陆：
```

Comment:
```suggestion
用 `rw_user1@localhost` 登陆。TiDB 中 无法通过 `localhost` 登陆，需要在 tidb-server 机器上指定 `127.0.0.1` 后才可登陆：
```

Reply:
> TiDB 中 无法通过 `localhost` 登陆

No, TiDB can use `localhost`, you need to configure the unix domain socket by setting [here](https://github.com/pingcap/tidb/blob/3ed4a23b956da63c95a5665432fef687a096601c/config/config.toml.example#L19)

```
host = ""
port = 0
socket = "/var/run/mysqld/mysqld.sock"
```

If you modify those configure items, you can do use localhost.


Or maybe you can update some of the related content to `127.0.0.1` if you find this guide confuse you.